client/src/tokens/typography.js

```diff
@@ -72,6 +72,7 @@ const fontFamily = {
  * These values are used in combinations create typography defaults
  */
 const fontSize = {
+  12: '0.75rem',
```

Comment:
We’ve worked hard so far to make sure to avoid small text so I’d prefer we don’t do this without chatting with @benenright again. If we really must have tiny text for the filters, we can use arbitrary / hard-coded values.

Reply:
Discussed with Sage: will remove, re-add later if needed.